.github/workflows/cmake.yml

```
@@ -57,6 +57,26 @@ jobs:
           cmake . -DCMAKE_PREFIX_PATH=/opt/jwt-cpp/cmake -DTEST:STRING="defaults-enabled"
```

Assuming we change the path this should now work

```suggestion
          cmake . -DCMAKE_PREFIX_PATH="/opt/jwt-cpp" -DTEST:STRING="defaults-enabled"
```